src/NHSD.BuyingCatalogue.Identity.Api/Controllers/AccountController.cs

```diff
@@ -163,9 +163,11 @@ public IActionResult ForgotPasswordLinkSent()
         }
 
         [HttpGet]
-        public IActionResult ResetPassword(string email, string token)
+        public async Task<IActionResult> ResetPassword(string email, string token)
```

Comment:
I've just tried applying this change but I think we're probably better leaving it as is because the view names also need to change and I don't think it makes much sense for a view to have the `Async` suffix.

Perhaps we need to tweak our standards so that it doesn't apply to MVC controllers (obviously fine in API controllers though)...

Reply:
👍 Saying that we have had a few issues with ending controller methods with Async. See the TrimAsync method, but yeah agreed one to review & makes sense to leave it out here.